README.adoc

```diff
@@ -15,17 +15,17 @@
 :page-related-guides: ['rest-intro', 'cdi-intro', 'microprofile-config', 'circuit-breaker']
 :common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/master
 :source-highlighter: prettify
-= Building fault tolerant microservices with the @Retry and @Fallback annotations
+= Building fault-tolerant microservices with the @Retry and @Fallback annotations
 
-Learn how to add retry and fallback behavior to microservice dependencies to manage the impact of failures.
+You'll explore how to manage the impact of failures by adding retry and fallback behaviors to microservice dependencies.
```

Comment:
The latest abstract for this guide is determined by YK to be: Learn how to add retry and fallback behavior to microservice dependencies to manage the impact of failures using MicroProfile Fault Tolerance. I think changing "Learn" to "You'll explore" is ok. @yeekangc can you help review this ID edit? Thanks.

Reply:
I am fine with this.

I have noticed such change/edit in other guide(s) too.  We should take a look at all the guides and ensure consistency.  This can be a separate issue.

Reply:
Will do. Thanks.